integration/client/restart_monitor_linux_test.go

```diff
@@ -78,9 +78,14 @@ version = 2
 		t.Fatal(err)
 	}
 
-	task.Kill(ctx, syscall.SIGKILL)
+	if err := task.Kill(ctx, syscall.SIGKILL); err != nil {
+		t.Fatal(err)
+	}
+
 	begin := time.Now()
-	deadline := begin.Add(interval).Add(epsilon)
+
+	// Give enough time to see whether a failure is a timing issue or not.
+	deadline := begin.Add(interval * 2)
```

Comment:
suggest interval + epsilon*count ... looping on sleep `epsilon` and check status count times where count is set to 10 above as a const

Reply:
Yes. I do agree that `interval * 2` is too arbitrary. Updated on the last revision.